ApplicationLibCode/GeoMech/GeoMechVisualization/RivFemPartGeometryGenerator.cpp

```diff
@@ -81,7 +81,7 @@ ref<DrawableGeo> RivFemPartGeometryGenerator::generateSurface( const std::vector
 //--------------------------------------------------------------------------------------------------
 ref<DrawableGeo> RivFemPartGeometryGenerator::createMeshDrawable()
 {
-    if ( !( m_quadVertices.notNull() && m_quadVertices->size() != 0 ) ) return nullptr;
+    if ( !m_quadVertices.notNull() || m_quadVertices->size() == 0 ) return nullptr;
```

Comment:
Same here.

Reply:
This is a cvf::vector, empty() is not implemented